public/components/metrics/redux/slices/metrics_slice.ts

```diff
@@ -111,18 +123,49 @@ const fetchRemoteMetrics = async (remoteDataSources: string[]): Promise<any> =>
   );
 };
 
+export const fetchOpenTelemetryIndices = async () => {
+  const { http } = coreRefs;
+  return http
+    .get(`${OBSERVABILITY_BASE}/search/indices`, {
+      query: {
+        format: 'json',
+      },
+    })
+    .catch((error) => console.error(error));
+};
+
+export const fetchOpenTelemetryDocumentNames = (selectedOtelIndex: string) => async () => {
+  const { http } = coreRefs;
+  return http
+    .post(`${OBSERVABILITY_BASE}/metrics/otel/documentNames`, {
+      body: JSON.stringify({
+        index: selectedOtelIndex,
+      }),
+    })
+    .catch((error) => console.error(error));
+};
+
 const updateLayoutBySelection = (state: any, newMetric: any) => {
   const newDimensions = getNewVizDimensions(state.metricsLayout);
 
+  const metricCatalog = (catalog: string) => {
```

Comment:
This catalog methodology is largely gone.  We'll have to look at how you're using this in conetxt of the new metrics.